docs/_data/errorref/meshery-server_errors_export.json

```diff
@@ -655,10 +655,10 @@
         "name": "ErrCompConfigPairsCode",
         "code": "1073",
         "severity": "Alert",
-        "long_description": "",
-        "short_description": "unable to Create Comp Config.",
-        "probable_cause": "",
-        "suggested_remediation": ""
+        "long_description": "An error occurred while trying to create Component Configuration.",
+        "short_description": "Unable to Create Component Configuration.",
+        "probable_cause": "Missing or invalid user token, invalid Kubernetes handler or serviceActionProvider error.",
```

Comment:
Refactor to make this user-facing. To a user, what is a Kubernetes handler or serviceActionProvider? 

Users know about Meshery Connections and can be offered a link to troubleshooting or understanding connections and Meshery's interactions with Kubernetes (in Meshery Docs), but they will not understand references to function names or golang routines.